packages/ove-app-svg/src/client/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Is this tag required? This is no longer required for Edge, only for older browsers like IE 9 which I presume are unsupported in Chrome anyway

Reply:
Nope, this is no longer required. Took me around 15mins to read and understand [the wonderful world of IE and how this tag works](https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do). It seems these tags could also be injected by the webservers (if we ever need it at a later date).

Fixed in 8357c37 for all ove-apps, and the same should also be fixed in OVE Core (see ove/ove#11.